packages/eslint-plugin-svelte/src/rules/no-unused-class-name.ts

```diff
@@ -57,7 +58,17 @@ export default createRule('no-unused-class-name', {
 						? findClassesInPostCSSNode(styleContext.sourceAst, sourceCode.parserServices)
 						: [];
 				for (const className in classesUsedInTemplate) {
-					if (!allowedClassNames.includes(className) && !classesUsedInStyle.includes(className)) {
+					if (
+						!allowedClassNames.includes(className) &&
+						!allowedClassNames.some((allowedClassName: string) => {
+							if (!isRegExp(allowedClassName)) {
+								return false;
+							}
+
+							return toRegExp(allowedClassName).test(className);
+						}) &&
+						!classesUsedInStyle.includes(className)
```

Comment:
I think this is simpler?

```suggestion
						allowedClassNames.none((allowedClassName) => {
							if (!isRegExp(allowedClassName)) {
								return allowedClassName === className;
							}

							return toRegExp(allowedClassName).test(className);
						}) &&
						!classesUsedInStyle.includes(className)
```

Reply:
Yes, I think this code is enough. (All Test cases passed)

https://github.com/sveltejs/eslint-plugin-svelte/pull/1257/commits/06cbf3fdbd1004b026ef0c9ca5d70b4f19728a2d